tests/e2e/specs/modules/analytics/setup-with-account-with-tag.test.js

```diff
@@ -134,28 +123,19 @@ describe( 'setting up the Analytics module with an existing account and existing
 		await resetSiteKit();
 	} );
 
-	it( 'pre-selects account and property if an existing tag is found that matches one belonging to the user and prevents them from being changed', async () => {
+	it( 'pre-selects account and property if an existing tag is found that matches one belonging to the user', async () => {
```

Comment:
This reads incorrectly - an existing tag should no longer affect which property is preselected, particularly because the user may not have access to it which we no longer check.

The matched property is based purely on properties the user has access to that are matched by URL. This could result in the same property of the existing tag being matched but the existing tag does not cause it.

https://github.com/google/site-kit-wp/blob/d0b94fd567eefa71259109c08b04fa24e04e3ab0/assets/js/modules/analytics/datastore/properties.js#L275-L284